python/docs/source/getting_started/index.rst

```diff
@@ -20,3 +20,13 @@
 Getting Started
 ===============
 
+**PySpark** is the Python API for Spark.
+
+This page lists an overview of the basic steps required to setup & get started with PySpark.
+
+.. toctree::
+   :maxdepth: 2
+
+   installation
+   package_overview
```

Comment:
Let's remove `package_overview`, `quickstart` and the corresponding RST files. It's always better to leave a PR focused on one issue without other changes.

Reply:
Done